doc/posts/2023/2023_07_27_Shilpi_week8.rst

```diff
@@ -0,0 +1,33 @@
+Generating Fit Functions : Week 9
```

Comment:
PR say week 8+9
filename say week8
title say week 9

Can you update that and make it more consistent.....